gc/gc-base/src/main/java/org/projectnessie/gc/base/IdentifyContentsPerExecutor.java

```diff
@@ -109,9 +109,7 @@ private IdentifiedResult computeExpiredContents(
         StreamingUtil.getCommitLogStream(
             gcStateParamsPerTask.getApi(),
             Detached.REF_NAME,
```

Comment:
With `ref` param being of type `Reference`, this could be simplified to `Detached.of(<hash>)` and the consumer can be `null`